src/com/cft/contactmerge/tests/CompareContactPartsTest.java

```diff
@@ -7,7 +7,7 @@
 class CompareContactPartsTest {
 
     @Test
-    void doNamesMatch_NoDifferentLastName() {
+    void doNamesMatch_No_DifferentLastName() {
```

Comment:
Good idea. I like the extra underscore. Let me know if you want me to change the rest or if you are going to do it.